ciphers/ColumnarTranspositionCipher.java

```diff
@@ -1,168 +1,198 @@
+     * Transposition Cipher Rule
+     *
+     * @return An Object[][] with the word to be encrypted filled in rows and
+     * columns
      */
-    private static Object[][] tableBuilder(String word){
-        Object[][] table=new Object[rows(word)+1][keyword.length()];
-        char[] wordInChards=word.toCharArray();
+    private static Object[][] tableBuilder(String word) {
+        Object[][] table = new Object[numberOfRows(word) + 1][keyword.length()];
+        char[] wordInChards = word.toCharArray();
         //Fils in the respective numbers
-        table[0]=findElements();
-        int charElement=0;
-        for(int i=1;i<table.length;i++){
-            for(int j=0;j<table[i].length;j++){
-                if(charElement<wordInChards.length){
-                    table[i][j]=(char)wordInChards[charElement];
+        table[0] = findElements();
+        int charElement = 0;
+        for (int i = 1; i < table.length; i++) {
+            for (int j = 0; j < table[i].length; j++) {
+                if (charElement < wordInChards.length) {
+                    table[i][j] = wordInChards[charElement];
                     charElement++;
-                }else{
-                    table[i][j]=ENCRYPTION_FIELD_CHAR;
+                } else {
+                    table[i][j] = ENCRYPTION_FIELD_CHAR;
                 }
             }
         }
         return table;
     }
+
     /**
-     * Determines the number of rows the table should have regarding the Columnar Transposition Cipher Rule
-     * @return an int with the number of rows that the table should have in order to respect the Columnar Transposition Cipher Rule.
+     * Determines the number of rows the table should have regarding the
+     * Columnar Transposition Cipher Rule
+     *
+     * @return an int with the number of rows that the table should have in
+     * order to respect the Columnar Transposition Cipher Rule.
      */
-    private static int rows(String word){
-        if((double)word.length()/keyword.length()>word.length()/keyword.length()){
-            return (word.length()/keyword.length())+1;
-        }else{
-            return word.length()/keyword.length();
+    private static int numberOfRows(String word) {
+        if ((double) word.length() / keyword.length() > word.length() / keyword.length()) {
+            return (word.length() / keyword.length()) + 1;
+        } else {
+            return word.length() / keyword.length();
         }
     }
-    private static Object[] findElements(){
-        Object[] charValues=new Object[keyword.length()];
-        for(int i=0;i<charValues.length;i++){
-            for(int j=0;j<abecedarium.length();j++){
-                if(keyword.charAt(i)==abecedarium.charAt(j))charValues[i]=j;
+
+    private static Object[] findElements() {
+        Object[] charValues = new Object[keyword.length()];
+        for (int i = 0; i < charValues.length; i++) {
+            for (int j = 0; j < abecedarium.length(); j++) {
```

Comment:
This loop can be avoid with help of the method ```.indexOf(c)``` [see](https://www.tutorialspoint.com/java/java_string_indexof.htm)

Reply:
:+1: